crypto/ec/ecdsa_aid.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdlib.h>
+
+#include <openssl/objects.h>
+#include "crypto/ec.h"
+
+#define ASN1_SEQUENCE 0x30
+#define ASN1_OID 0x06
+#define OID_FIRST(a, b) a * 40 + b
+#define DER_840() 0x86, 0x48    /* DER encoding of number 840 is 2 bytes */
```

Comment:
Hi @slontis , how did you get these numbers? By checking the OID in a particular certificate file for instance?

Reply:
Multiple ways..
This is just the DER encoding of any value > 127 as described here..
https://www.sciencedirect.com/topics/computer-science/basic-encoding-rules
It is also listed in mutliple online OID repositories as well as openssl's table of OIDS.

Reply:
Great, thanks